static/include/components/servo-control.md

```diff
@@ -0,0 +1,7 @@
+## Test the servo
+
+After setting up and establishing the connection for your servo, navigate to the [Control tab](https://docs.viam.com/manage/fleet/robots/#control) and select the dropdown panel dedicated to the motor.
```

Comment:
```suggestion
After setting up and establishing the connection for your servo, navigate to the [Control tab](/manage/fleet/robots/#control) and select the dropdown panel dedicated to the motor.
```

You can use a relative link here. Then on staging - the link that get's commented on a pull request it will link to the page in the test environment - which helps with reviews